clients/turbo.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": [".next/*", "!.next/cache/*"]
+    },
+    "dev": {
+      "dependsOn": ["build"],
```

Comment:
I don't think you need to `npm run build` before doing an `npm run dev`, you should only need `npm run install`—or am I misinterpreting how this config works?

Reply:
I think you're right, I'll remove

Reply:
this config basically says the `build` pipeline step needs to run for all applicable workspaces before any `dev` commands run